demo/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Apply or discuss the same changes as in the main conf file

Reply:
What do you think will be better?
To copy same configs from **main** to **demo**, so it will become clearer?
Or demo `tsconfig.json` must extend the **main** one?

Reply:
I think **demo** can extend the **main** one